samcli/commands/init/__init__.py

```diff
@@ -1,404 +1,4 @@
-            if not app_template:
-                if len(options) == 1:
-                    app_template = options[0].get("appTemplate")
-                elif len(options) > 1:
-                    raise LambdaImagesTemplateException(
-                        "Multiple lambda image application templates found. "
-                        "Please specify one using the --app-template parameter."
-                    )
-
-        if app_template and not location:
-            location = templates.location_from_app_template(
-                package_type, runtime, base_image, dependency_manager, app_template
-            )
-            no_input = True
-        extra_context = _get_cookiecutter_template_context(name, runtime, architecture, extra_context)
-
-        if not output_dir:
-            output_dir = "."
-        do_generate(
-            location,
-            package_type,
-            runtime,
-            dependency_manager,
-            output_dir,
-            name,
-            no_input,
-            extra_context,
-            tracing,
-        )
-    else:
-        if not (pt_explicit or runtime or dependency_manager or base_image or architecture):
-            click.secho(INIT_INTERACTIVE_OPTION_GUIDE, fg="yellow", bold=True)
-
-        # proceed to interactive state machine, which will call do_generate
-        do_interactive(
-            location,
-            pt_explicit,
-            package_type,
-            runtime,
-            architecture,
-            base_image,
-            dependency_manager,
-            output_dir,
-            name,
-            app_template,
-            no_input,
-            tracing,
-        )
-
-
-def _deprecate_notification(runtime):
-    from samcli.lib.utils.colors import Colored
-
-    if runtime in DEPRECATED_RUNTIMES:
-        message = (
-            f"WARNING: {runtime} is no longer supported by AWS Lambda, please update to a newer supported runtime. "
-            "For more information please check AWS Lambda Runtime Support Policy: "
-            "https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html"
-        )
-        LOG.warning(Colored().yellow(message))
-
-
-def _get_cookiecutter_template_context(name, runtime, architecture, extra_context):
-    default_context = {}
-    extra_context_dict = {}
-
-    if runtime is not None:
-        default_context["runtime"] = runtime
-
-    if name is not None:
-        default_context["project_name"] = name
-
-    default_context["architectures"] = {"value": get_architectures(architecture)}
-    if extra_context is not None:
-        try:
-            extra_context_dict = json.loads(extra_context)
-        except JSONDecodeError as ex:
-            raise click.UsageError(
-                "Parse error reading the --extra-context parameter. The value of this parameter must be valid JSON."
-            ) from ex
-
-    return {**extra_context_dict, **default_context}
+from .command import cli
```

Comment:
👏🏼 I was just checking another thing in `sam init` and this file really bothered me. Thanks for making it aligned with other commands!